### DISCUSSION TITLE
How to sign back to the google account that I have changed password

### DISCUSSION THREAD

**Discussion by NiceAssists** [TeamVanced/VancedManager#732] [Q&A]
I have changed my google acc password. When I entered youtube vanced, it shows that there are no internet connections, but my device is connected to the internet. I try to relog into the account but I cant find the option to do it. Does anyone have any solutions?

**Comment by jessechumo** [comment]
Usually works for me after uninstalling and re-installing the micro-g app. Also, try clearing youtube vanced data and cache if the problem persists.
